backend/src/main/scala/cromwell/backend/io/WorkflowPathsBackendInitializationData.scala

```diff
@@ -0,0 +1,51 @@
+package cromwell.backend.io
+
+import java.nio.file.FileSystem
+
+import cromwell.backend.BackendInitializationData
+
+/**
+  * Extension of backend initialization data that also provides a `WorkflowPaths`, and by proxy its `List[FileSystem]`.
+  *
+  * This class consolidates code used by the legacy Local backend and the SFS backend, that need to localize data
+  * from a collection of different file systems (local and GCS for now).
+  *
+  * NOTE: The JES backend uses a fork of the `WorkflowPaths`. As `JesWorkflowPaths` is not a subclass, the
+  * initialization data for JES does not use this trait nor the companion object. It is not clear if JES needs the
+  * functionality, as it cannot localize from generic file systems, only GCS with different authentication modes.
```

Comment:
Can we ticket or TODO looking into unifying `WorkflowPath` with `JesWorkflowPaths`?